functests/utils/consts.go

```diff
@@ -20,6 +26,22 @@ func init() {
 	if PerformanceProfileName == "" {
 		PerformanceProfileName = "performance"
 	}
+
+	if discovery.Enabled() {
```

Comment:
I'd rather move this logic in the discovery package and use it from here.
It doesn't feel it belongs to `const.go`

Reply:
moved previous logic to discovery.go